src/graphql/resolvers.ts

```diff
@@ -251,7 +251,20 @@ const resolvers = {
       const { media }: { media: MediaDataSource } = dataSources
       return await media.findMediaByAreaId(node.metadata.area_id, node.ancestors)
     },
-    authorMetadata: getAuthorMetadataFromBaseNode
+
+    authorMetadata: getAuthorMetadataFromBaseNode,
+
+    organizations: async (node: AreaType, args: any, { dataSources }: Context) => {
+      const { organizations } = dataSources
+      const areaIdsToSearch = [node.metadata.area_id, ...node.ancestors.split(',').map(s => muid.from(s))]
+      const associatedOrgs = await (await organizations.findOrganizationsByFilter({
```

Comment:
In [`findOrganizationsByFilter()`](https://github.com/OpenBeta/openbeta-graphql/blob/17ba6ad6dabbc6e5b5e9cb725768cc81085f7f83/src/model/OrganizationDataSource.ts#LL43C33-L43C44) maybe add `toArray()` there and await to avoid 2 awaits here?

Reply:
Thanks! Good suggestion. A lot cleaner that way :)

Reply:
Hmm unfortunately there's another reference that requires the endpoint to return a cursor instead of array:
<img width="778" alt="image" src="https://github.com/OpenBeta/openbeta-graphql/assets/3641356/bfd40eff-3612-4357-bc9a-00ec1395edad">

I tried to clean it up and split out the awaits instead. Hopefully a bit more readable. Lmk what you think.